### PR TITLE
MC-9575 Add plugin page for OpenID Connect

### PR DESCRIPTION
**Pull request by pjmonks**
Covers technical aspect of how to use the OpenID Connect plugin, around using endpoints.

### CODE REVIEW

**Review by jamesrwelch — COMMENTED** [review]
I like this a lot.  I won't merge it until the plugin is officially approved / released

File: docs/plugins/openid-connect.md
Comment (olliefreeman) [inline docs/plugins/openid-connect.md:11]: does the relevant pieces of information exist at this page??

**Review by olliefreeman — CHANGES_REQUESTED** [review]
We will want some additional stuff from me regarding configuring details via config to help bootstrap the providers we know about.

Do we have anything yet regarding configuring docker??? (yes i probably should know this but i dont)

File: docs/plugins/openid-connect.md
Comment (pjmonks) [reply docs/plugins/openid-connect.md:11]: The user guide for Open ID connect is pending, notes on what it should include have been forwarded to Gemma (and @jamesrwelch was CC'd in too).

**Pull request merged**
